packages/core/hooks/precompact-dump.py

```
@@ -0,0 +1,199 @@
+                "precompact",
+                {
+                    "trigger": trigger,
+                    "dump_path": os.path.relpath(dump_path, project_dir),
+                    "bytes": len(text),
+                    "has_plans": bool(plans_text),
+                },
+                session_id=session_id,
+                project_dir=project_dir,
+            )
+        except Exception:  # pragma: no cover
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)